src/jq.test.js

```diff
@@ -40,18 +37,6 @@ describe('jq core', () => {
       })
   })
 
-  it('should pass on an empty filter', done => {
-    run('', PATH_JSON_FIXTURE)
-      .then(output => {
-        const normalizedOutput = output.replace(/\r\n/g, '\n')
-        expect(normalizedOutput).to.equal(FIXTURE_JSON_STRING)
-        done()
-      })
-      .catch(error => {
-        done(error)
-      })
-  })
```

Comment:
Does jq stopped supporting this?

Reply:
Yes, jq dropped supporting empty query; 
```
 $ jq -n ''
jq: error: Top-level program not given (try ".")
jq: 1 compile error
```
See https://github.com/jqlang/jq/issues/2785.